tools/topology/topology2/include/pipelines/cavs/gain-capture.conf

```diff
@@ -69,12 +69,26 @@ Class.Pipeline."gain-capture" {
 				out_bit_depth		32
 				out_valid_bit_depth	32
 			}
+			# 32-bit 48KHz 4ch
+			Object.Base.audio_format.4 {
```

Comment:
@RanderWang please use the input_audio_format and output_audio_format objects explicitly

Reply:
NP. Thanks!